src/Controls/tests/DeviceTests/Elements/Accessibility/AccessibilityTests.cs

```diff
@@ -0,0 +1,208 @@
+﻿#if !WINDOWS
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Microsoft.Maui.Handlers;
+using Xunit;
+using Microsoft.Maui.Controls.Hosting;
+using Microsoft.Maui.Controls;
+using Microsoft.Maui.Hosting;
+using Microsoft.Maui.Controls.Shapes;
+using System.Text;
+using Microsoft.Maui.DeviceTests.Stubs;
+
+namespace Microsoft.Maui.DeviceTests
+{
+	[Category(TestCategory.Accessibility)]
+	public partial class AccessibilityTests : ControlsHandlerTestBase
+	{
+		[Theory("IsInAccessibleTree initializes correctly")]
+		[InlineData(true)]
+		[InlineData(false)]
+		public async Task IsInAccessibleTree(bool result)
+		{
+			var button = new Button();
+			AutomationProperties.SetIsInAccessibleTree(button, result);
+			var important = await GetValueAsync<bool, ButtonHandler>(button, handler => button.IsAccessibilityElement());
+			Assert.Equal(result, important);
+		}
+
+#if !WINDOWS
+		[Theory("ExcludedWithChildren initializes correctly")]
+		[InlineData(true)]
+		[InlineData(false)]
+		public async Task ExcludedWithChildren(bool result)
+		{
+			var button = new Button();
+			AutomationProperties.SetExcludedWithChildren(button, result);
+			var excluded = await GetValueAsync<bool, ButtonHandler>(button, handler => button.IsExcludedWithChildren());
+			Assert.Equal(result, excluded);
+		}
+#endif
+
+		[Category(TestCategory.Accessibility)]
+		[Collection(ControlsHandlerTestBase.RunInNewWindowCollection)]
+		public class InNewWindowCollection : ControlsHandlerTestBase
+		{
+			[Fact]
+			public async Task ValidateIsImportantForAccessibility()
+			{
+				EnsureHandlerCreated(builder =>
+				{
+					builder.ConfigureMauiHandlers(handlers =>
+					{
+						handlers.AddMauiControlsHandlers();
+						handlers.AddHandler(typeof(Window), typeof(WindowHandlerStub));
+					});
+				});
+
+				// Ideally this would use a theory and class data for each control type.
+				//
+				// To accurately test this we need to attach the elements to the UI Tree.
+				// Currently tests that attach to the UI Tree can't run in parallel so it slows everyone down.
+				// So, I opted to just attach a layout and then iterate over the controls.
+				// I left the test case class in case it makes sense to switch in the future. 
+
+				var layout = new VerticalStackLayout();
+				var contentPage = new ContentPage()
+				{
+					Content = new ScrollView()
+					{
+						Content = layout
+					}
+				};
+
+				var expectedResults = new Dictionary<IView, bool>();
+
+				foreach (var control in new AllControlsTestCase())
+				{
+					var view = (IView)Activator.CreateInstance((Type)control[0]);
+
+					if (view is RefreshView rv)
+						rv.Content = new VerticalStackLayout();
```

Comment:
Why RefreshView and why only RefreshView here? (as opposed to also other layouts)

Reply:
Aw yea, let me remove this. I was working around a bug with `RefreshView` that was recently fixed https://github.com/dotnet/maui/pull/11735